hera_pspec/pspecdata.py

```diff
@@ -517,12 +552,15 @@ def set_R(self, R_matrix):
         R_matrix : string or matrix
             If set to "identity", sets R = I
             If set to "iC", sets R = C^-1
+            If set to "identity_with_flags", sets R = diagonal according to pspecdata weights
             Otherwise, accepts a user inputted dictionary
         """
         if R_matrix == "identity":
             self.R = self.I
         elif R_matrix == "iC":
             self.R = self.iC
+        elif R_matrix == "diagonal":
```

Comment:
the docstring above says `identity_with_flags` but what's actually caught here is `diagonal`.

Reply:
Whoops. Fixed the docstring.